solr/core/src/test/org/apache/solr/legacy/TestNumericRangeQuery32.java

```diff
@@ -183,12 +183,12 @@ private void testRange(int precisionStep) throws Exception {
       ScoreDoc[] sd = topDocs.scoreDocs;
       assertNotNull(sd);
       assertEquals("Score doc count" + type, count, sd.length);
-      Document doc = searcher.doc(sd[0].doc);
+      Document doc = searcher.storedFields().document(sd[0].doc);
```

Comment:
Here we have an `IndexSearcher` i.e. not a `SolrIndexSearcher` and so not using `SolrIndexSearcher.getDocFetcher()` here.